'publishToMavenLocal' task not found

## Description
<!-- Describe your issue -->
https://jitpack.io/#elimu-ai/appstore

https://jitpack.io/com/github/elimu-ai/appstore/2.4.0/build.log

```
FAILURE: Build failed with an exception.

* What went wrong:
Task 'publishToMavenLocal' not found in root project 'elimu.ai Appstore' and its subprojects.
```

## Why?
<!-- Explain _why_ this issue is needed -->
This fix is needed to automate APK release management through JitPack.

---

> elimu.ai's mission is to build innovative learning software that empowers out-of-school children to teach themselves basic reading📖, writing✍🏽 and math🔢 **within 6 months**.

> ## Description
> https://jitpack.io/#elimu-ai/appstore
> 
> https://jitpack.io/com/github/elimu-ai/appstore/2.4.0/build.log
> 
> ```
> FAILURE: Build failed with an exception.
> 
> * What went wrong:
> Task 'publishToMavenLocal' not found in root project 'elimu.ai Appstore' and its subprojects.
> ```
> 
> ## Why?
> This fix is needed to automate APK release management through JitPack.
> 
> > elimu.ai's mission is to build innovative learning software that empowers out-of-school children to teach themselves basic reading📖, writing✍🏽 and math🔢 **within 6 months**.

@jo-elimu Would that be ok if we attach the output APK to the build output of Github Actions check, as in here?
https://github.com/elimu-ai/filamu/pull/10

--- a/README.md
+++ b/README.md
@@ -1,5 +1,7 @@
 # elimu.ai Appstore 📲
 
+[![Release](https://jitpack.io/v/elimu-ai/appstore.svg)](https://jitpack.io/#elimu-ai/appstore)
+
 Android application which downloads and installs educational apps stored on the [elimu.ai](http://elimu.ai) platform.
 
 <img width="320" src="https://user-images.githubusercontent.com/15718174/84632262-39fec180-af21-11ea-8a8a-215120744f05.png">

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -12,8 +12,8 @@ android {
         minSdkVersion 26
         compileSdk 35
         targetSdkVersion 35
-        versionCode 2004001
-        versionName "2.4.1"
+        versionCode 2004002
+        versionName "2.4.2-SNAPSHOT"
 
         setProperty("archivesBaseName", "${applicationId}-${versionCode}")
 

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -85,3 +85,6 @@ publishing {
         }
     }
 }
+tasks.named("publishApkPublicationToMavenLocal") {
+    mustRunAfter(":app:assembleRelease")
+}

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -13,7 +13,7 @@ android {
         compileSdk 35
         targetSdkVersion 35
         versionCode 2004001
-        versionName "2.4.1-SNAPSHOT"
+        versionName "2.4.1"
 
         setProperty("archivesBaseName", "${applicationId}-${versionCode}")
 

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -12,8 +12,8 @@ android {
         minSdkVersion 26
         compileSdk 35
         targetSdkVersion 35
-        versionCode 2004002
-        versionName "2.4.2"
+        versionCode 2004003
+        versionName "2.4.3-SNAPSHOT"
 
         setProperty("archivesBaseName", "${applicationId}-${versionCode}")
 

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -13,7 +13,7 @@ android {
         compileSdk 35
         targetSdkVersion 35
         versionCode 2004002
-        versionName "2.4.2-SNAPSHOT"
+        versionName "2.4.2"
 
         setProperty("archivesBaseName", "${applicationId}-${versionCode}")
 

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -1,5 +1,6 @@
 plugins {
     alias(libs.plugins.kotlin.android)
+    id 'maven-publish'
 }
 apply plugin: 'com.android.application'
 
@@ -75,4 +76,12 @@ dependencies {
 
     androidTestImplementation libs.androidx.junit
     androidTestImplementation libs.androidx.espresso
+}
+
+publishing {
+    publications {
+        apk(MavenPublication) {
+            artifact("${buildDir}/outputs/apk/release/${android.defaultConfig.applicationId}-${android.defaultConfig.versionCode}-release-unsigned.apk")
+        }
+    }
 }

